mlflow/utils/environment.py

```diff
@@ -175,7 +175,8 @@ def _is_iterable(x):
 
 
 _INFER_PIP_REQUIREMENTS_FALLBACK_MESSAGE = (
-    "Encountered an unexpected error while inferring pip requirements (model URI: %s, flavor: %s)"
+    "Encountered an unexpected error while inferring pip requirements (model URI: %s, flavor: %s),"
+    " fallback to generate default pip requirements."
```

Comment:
```suggestion
    "Encountered an unexpected error while inferring pip requirements (model URI: %s, flavor: %s), "
    "fall back to return %s. Set logging level to DEBUG to see the full traceback."
```

Reply:
I think it's ok to always show `Set logging level to DEBUG to see the full traceback.`.

Reply:
I removed `default pip requirements` since `fallback` might not be a default set of pip requirements.